### PR TITLE
Added method fillWithRequest

### PR DESCRIPTION
**Pull request by SolStis86**
Useful for store and update methods in controllers.  This may couple Eloquent to closely to Illuminate\Http

### CODE REVIEW

File: src/Illuminate/Database/Eloquent/Model.php [hunk 1]
```diff
@@ -251,6 +252,17 @@ public function forceFill(array $attributes)
     }
 
     /**
+     * Fill the model with fillable attributes present in a Request object.
+     *
+     * @param  Illuminate\Http\Request  $request
```
Comment (gabriel-caruso) [inline src/Illuminate/Database/Eloquent/Model.php]: Should be `\Illuminate\Http\Request`

**Review by carusogabriel — CHANGES_REQUESTED** [review]
Fix `@param` typo

**Comment by Dylan-DPC** [comment]
What's the use of this when you have `fill`?

**Comment by masterwto** [comment]
Do sth. useful

**Comment by SolStis86** [comment]
@Dylan-DPC your right its useless.. i was thinking along the lines of MassAssignmentExceptions but this would only occur on create rather than update / save.  

```$model->update($request->toArray());```

would work fine... closing

**Pull request closed without merging**